main.go

```diff
@@ -69,6 +69,7 @@ var cmds = []*cli.Command{
 				Usage: "[DEPRECATED] Specify a full image reference, e.g.: quay.io/some/image:tag",
 			},
 			&sourceFlag,
+			&cli.StringFlag{Name: "boot-entry", Usage: "Specify a systemd-boot entry to upgrade (other than active/passive/recovery). The value should match the name of the '.efi' file."},
```

Comment:
It doesn't affect the input the user gives, but mainly for consistency

Reply:
Hm given the value should match the efi file name, if one would try to upgrade active, the value would have been `active` which is what I'm trying to prevent here. But still, I don't know if it's more confusing this way. The "aliases" are not very helpful either. How would we describe the active in the example below?

<img width="377" alt="image" src="https://github.com/user-attachments/assets/51314519-18af-4ae5-8c65-2c4d01dbc2dc">

Reply:
Given it's just the command help message and that both alternatives are a bit confusing, I would leave it as it it until we have a better idea. What do you think?

Reply:
These are the entries I see after the installation when `--boot-branding` is set to "KairosUKI":

![image](https://github.com/user-attachments/assets/cdfd8934-1741-4da3-aaad-fb307fdbaa71)

I don't see `cos` anywhere.

Reply:
yeah that's what makes it even more confusing "cos" is not displayed, but it's the name used 😞

Reply:
we can always change it later so it's good for me